library/xml-model/src/main/java/org/cru/godtools/xml/model/tract/Card.kt

```diff
@@ -53,11 +53,10 @@ class Card : BaseModel, Styles, Parent {
     @get:ColorInt
     override val textColor get() = _textColor ?: page.cardTextColor
 
-    // TODO: implement card-background-color on Page & Manifest
     @ColorInt
     private val _backgroundColor: Int?
     @get:ColorInt
-    internal val backgroundColor get() = _backgroundColor ?: manifest.backgroundColor
+    internal val backgroundColor get() = _backgroundColor ?: page.cardBackgroundColor ?: manifest.cardBackgroundColor ?: manifest.backgroundColor
```

Comment:
because you have the page `card-background-color` fallback to the manifest `card-background-color` defined in `TractPage`, you can leave `manifest.cardBackgroundColor` out of this elvis operator chain